examples/basic-webpack/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+var path = require('path');
+
+module.exports = {
+  entry: './index.jsx',
+  output: {
+    path: path.resolve('./'),
+    filename: 'bundle.js'
+  },
+  module: {
+    loaders: [{
+      test: /\.jsx?$/,
+      exclude: /node_modules/,
+      loaders: ['react-hot', 'babel']
```

Comment:
React Hot Loader is experimental tech, it shouldn’t be here. (Additionally 1.x is deprecated.)